c/reverse/README.adoc

```diff
@@ -3,11 +3,49 @@
 DFINITY uses standard wasm binaries, so we may use standard tools such as Clang
 to build apps.
 
-First, install Clang and set up a couple of aliases. For example, on Debian:
+# Running in Docker
 
-  $ sudo apt-get install clang lld gcc-multilib
-  $ alias wld="wasm-ld-8 --no-entry --export-dynamic --allow-undefined"
-  $ alias wcc="clang --target=wasm32 -c -O3"
+To create a docker container with all the libraries required and dfx follow the below instructions.  If you already have a debian environment you'd like to use, skip to the following section.
+
+ Create a docker file with the below text:
+ 
+  FROM debian
+  RUN apt-get update
+  RUN apt-get install -y curl gnupg xz-utils nodejs npm wget git
+  RUN apt install -y lsb-release wget software-properties-common
```

Comment:
Are these required by llvm? dfx should be standalone and not depend on other packages. Pretty sure we don't need `nodejs` and `npm`, as there is no front end here.

Reply:
I had to install them to get some things to work.  Basically dfx build was failing when it was trying to build some things.  I can't find the error now, but it was right after the output of something like 'setting up the environment'.  My understanding was that node was required for dfx. Maybe that has changed in the latest release?  I'll try to replicate with out those packages and see.

Reply:
With the new dfx.json file, `dfx build` is simply copying wasm files around. It won't do any actual build.

Reply:
Your right! You don't need node for this example so I removed it. If you want to play around with any other dfx commands like new you do need it or you get:

Node could not be found. Skipping installing the frontend example code.

I've removed them for the time being.